src/components/CheckIn/CheckInItems.js

```diff
@@ -33,23 +38,25 @@ const ORDER_FORMAT_TO_PIECE_FORMAT = {
 
 class CheckInItems extends Component {
   static propTypes = {
+    addPiece: PropTypes.func.isRequired,
     checkedItems: PropTypes.arrayOf(PropTypes.object).isRequired,
     checkedItemsMap: PropTypes.object.isRequired,
     isAllChecked: PropTypes.bool.isRequired,
-    toggleAll: PropTypes.func.isRequired,
-    toggleItem: PropTypes.func.isRequired,
+    items: PropTypes.arrayOf(PropTypes.object).isRequired,
+    lineId: PropTypes.string.isRequired,
     location: ReactRouterPropTypes.location.isRequired,
     locations: PropTypes.arrayOf(PropTypes.object).isRequired,
-    addPiece: PropTypes.func.isRequired,
-    items: PropTypes.arrayOf(PropTypes.object).isRequired,
+    parentMutator: PropTypes.object.isRequired,
```

Comment:
please keep all business logic in Container component, `CheckInItemsWrapper`

Reply:
Updated